README.md

```diff
@@ -27,20 +27,20 @@ So, if you're serious about getting Astarte in your production environment, you
 
 Good question. This repository is a collection of utilities, home to Astarte's documentation and architecture decisions, and acts as an umbrella for the project. Astarte is a distributed system made up of several microservices, [which can all be found in Github](https://github.com/astarte-platform). Its core components are:
 
-* [Data Updater Plant](https://github.com/astarte-platform/astarte_data_updater_plant): Takes care of ingesting data into Cassandra and routing it to other Astarte components.
-* [Trigger Engine](https://github.com/astarte-platform/astarte_trigger_engine): Manages triggers, rules and push events - it is the component that delivers data to your application.
+* [Data Updater Plant](https://github.com/astarte-platform/astarte_data_updater_plant): Takes care of ingesting data into the database, filtering and routing it to other Astarte components.
+* [Trigger Engine](https://github.com/astarte-platform/astarte_trigger_engine): Processes incoming events, applies rules, prepares payloads and performs actions - it is the component that delivers data to your application.
 * [AppEngine API](https://github.com/astarte-platform/astarte_appengine_api): If you are building an application on top of Astarte's APIs, you will most likely call into it.
-* [Pairing](https://github.com/astarte-platform/astarte_pairing) & [Pairing API](https://github.com/astarte-platform/astarte_pairing_api): Keeps your device safe and secure. Device authentication and certificate orchestration happen here.
-* [Realm Management](https://github.com/astarte-platform/astarte_realm_management) & [Realm Management API](https://github.com/astarte-platform/astarte_realm_management_api): Where configuration happens. Manage your triggers, interfaces and more from here.
-* [Housekeeping](https://github.com/astarte-platform/astarte_housekeeping) & [Housekeeping API](https://github.com/astarte-platform/astarte_housekeeping_api): The *"superadmin"* component of Astarte: configure your global instance, manage realms and more.
+* [Pairing](https://github.com/astarte-platform/astarte_pairing) & [Pairing API](https://github.com/astarte-platform/astarte_pairing_api): Issues SSL certificates to the devices so they can connect to the broker.
```

Comment:
This isn't necessary true - certificates might be used against any Astarte transport, so it might be misleading. s/broker/transport/

Reply:
Also, Pairing takes care of more than that (e.g.: Agent), so I'd keep at least the certificate orchestration part, also because strictly speaking the CA emits certificates - not Pairing, and such a statement is highly misleading. Thinking about it twice, I don't really like your definition at all

Reply:
I wouldn't underline the agent part here. I would discourage this kind of scenario, so I don't like to point this here.

Reply:
Then I'd stay generic, and probably my definition works better - the CA part is quite misleading